server/models.py

```diff
@@ -0,0 +1,72 @@
+from app import db
+from sqlalchemy.sql import func
+from sqlalchemy import ForeignKey
+
+class contest_user(db.Model):
+    __tablename__ = 'contest_user'
+
+    id = db.Column(db.Integer, primary_key=True)
+    email = db.Column(db.String())
+    password = db.Column(db.String())
+    createdTime = db.Column(db.DateTime, default=func.utcnow())
+
+    def __init__(self, id, email, password, createdTime):
+        self.id = id
+        self.email = email
+        self.password = password
+        self.createdTime = createdTime
+
+class submission_user(db.Model):
```

Comment:
What is this model used for? what would be the difference between a submission user and a regular user?